packages/@aws-cdk/aws-cloudtrail/test/cloudtrail.test.ts

```diff
@@ -102,6 +103,31 @@ describe('cloudtrail', () => {
       expect(stack).not.toHaveResource('AWS::Logs::LogGroup');
     });
 
+    test('with sns topic', () => {
+      const stack = getTestStack();
+      const topic = new sns.Topic(stack, 'Topic');
+
+
+      new Trail(stack, 'Trail', { snsTopic: topic});
```

Comment:
were you leaving space here for //GIVEN //WHEN //THEN comments?
I think we added those for the most recent tests, but I'd encourage that as it helps with readability (for me at least)